client/src/plugins/privacy-preferences/PrivacyPreferences.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React, { PureComponent } from 'react';
+
+import PrivacyPreferencesView from './PrivacyPreferencesView';
+
+const CONFIG_KEY = 'editor.privacyPreferences';
+
+export default class PrivacyPreferences extends PureComponent {
```

Comment:
The idea here would be that this component considers the value of FLAGS(SERVER_INTERACTION) and becomes a noop.

Reply:
Thanks for the info. IMO it's better not to instantiate the class at all if the server-interaction flag is non-existent. Maybe if we really want to have less code in App.js, we can do some sort of early return inside PrivacyPreferences class depending on the flag, but I see no other benefit of doing that.

Reply:
I was thinking about doing the same for UpdateCheck module, which I also want to implement as a plugin.

Reply:
A benefit I can think of having certain plugins filtered out inside the App is that, in that case the render/constructor/mount methods of these plugins are not executed at all.